packages/application-generic/src/usecases/select-integration/select-integration.usecase.ts

```diff
@@ -57,7 +63,7 @@ export class SelectIntegration {
 
     let integration: IntegrationEntity | null = null;
 
-    if (command.filterData.tenant) {
+    if (command.identifier) {
```

Comment:
@davidsoderberg I think here it should still be the check that tenant is included in the command. 
What I meant in my previous comment about the integrationIdentifier override  (which is command.identifier here) is that I think it should take precedence over the tenant filters. I mean that if someone used the integration identifier override - that one should be used and not by the tenant. 
Right now, you are doing that check in the `getPrimaryIntegration` and only if the filters for tenant didn't find an integration